pattern-2/tile/tile.yml

```diff
@@ -63,12 +89,25 @@ forms:
   label: AM Analytics Cluster DB connection information
   description: WSO2 API Manager - Analytics Clustering Database connection information
   properties:
-  - name: sp_db_hostname
+  - name: sp_db_jdbc_url
     type: string
-    label: Hostname
-  - name: sp_db_name
+    label: JDBC URL
+  - name: sp_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+    - name: postgres
+      label: postgres
+    - name: oracle.jdbc.driver.OracleDriver
```

Comment:
Shall we remove this as well? We can add it when bosh deployment supports it.

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c